src/utils/FileDownload.ts

```diff
@@ -0,0 +1,16 @@
+export const fileDownload = (data: string | ArrayBuffer | ArrayBufferView | Blob, filename: string, mime?: string) => {
+    const blob = new Blob([data], { type: mime || 'application/octet-stream' });
+    const blobUrl = URL.createObjectURL(blob);
+
+    const a = document.createElement('a');
+    a.style.display = 'none';
+    a.download = filename;
+    a.href = blobUrl;
+
+    document.body.appendChild(a);
+
+    Promise.resolve(a.click()).then(() => {
+        document.body.removeChild(a);
+        URL.revokeObjectURL(blobUrl);
```

Comment:
A11Y guides I've followed in the past suggest leaving the link so that a screen reader doesn't loose context after the link is clicked.  Did you have a specific need to remove the link?

Reply:
I have a better appreciation of what's happening here now.

I'm almost wondering if it doesn't make sense to just refactor this code into DownloadLink to work with the element there so it can be refocused when the download is complete.

A different discussion (outside of this PR) might be switching to using signed URLs since the documents are already stored in GCS.

Reply:
We should revisit how we do downloads to make it consistent across the UI, but to do that I think we would need to do a larger evaluation of all the different ways we do downloads more generally. In the short term my goal is to deprecate the use of this package.